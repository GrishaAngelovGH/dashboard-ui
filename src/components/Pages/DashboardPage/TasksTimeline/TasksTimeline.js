import { Timeline } from 'antd'
import { ClockCircleOutlined } from '@ant-design/icons'

const TasksTimeline = () => (
    <Timeline mode='alternate'>
        <Timeline.Item>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, nisi?</Timeline.Item>
        <Timeline.Item color='green'>Lorem ipsum dolor sit amet.</Timeline.Item>
        <Timeline.Item dot={<ClockCircleOutlined style={{ fontSize: '16px' }} />}>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur eaque quam, aperiam iste vel ipsum laudantium debitis ratione maxime voluptates sequi reprehenderit repellendus rerum nihil distinctio exercitationem vero inventore magni non. Ipsam ipsum maiores consequatur cupiditate excepturi, neque quam dolor facere quos magnam ea sed maxime expedita vitae nobis aliquam?
        </Timeline.Item>
        <Timeline.Item color='red'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati, vel!</Timeline.Item>
        <Timeline.Item>Lorem ipsum, dolor sit amet consectetur adipisicing elit. At assumenda id eius. Cupiditate natus reprehenderit sint corrupti dolore provident molestias.</Timeline.Item>
        <Timeline.Item dot={<ClockCircleOutlined style={{ fontSize: '16px' }} />}>
            Lorem ipsum dolor sit amet.
        </Timeline.Item>
    </Timeline>
)

export default TasksTimeline