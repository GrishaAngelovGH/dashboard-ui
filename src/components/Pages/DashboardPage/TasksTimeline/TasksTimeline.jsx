import { Timeline } from 'antd'
import { ClockCircleOutlined } from '@ant-design/icons'

const TasksTimeline = () => (
	<Timeline mode='alternate' items={[
		{ children: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, nisi?' },
		{ children: 'Lorem ipsum dolor sit amet.', color: 'green' },
		{
			children: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur eaque quam, aperiam iste vel ipsum laudantium debitis ratione maxime voluptates sequi reprehenderit repellendus rerum nihil distinctio exercitationem vero inventore magni non. Ipsam ipsum maiores consequatur cupiditate excepturi, neque quam dolor facere quos magnam ea sed maxime expedita vitae nobis aliquam?',
			dot: <ClockCircleOutlined style={{ fontSize: '16px' }} />
		},
		{ children: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati, vel!', color: 'red' },
		{ children: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. At assumenda id eius. Cupiditate natus reprehenderit sint corrupti dolore provident molestias.' },
		{
			children: 'Lorem ipsum dolor sit amet.',
			dot: <ClockCircleOutlined style={{ fontSize: '16px' }} />
		}
	]} />
)

export default TasksTimeline