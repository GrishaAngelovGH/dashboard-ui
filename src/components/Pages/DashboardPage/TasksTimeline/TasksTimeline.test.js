import { render } from '@testing-library/react'

import TasksTimeline from './TasksTimeline'

test('should render component', () => {
    const container = render(<TasksTimeline />)

    expect(container).toMatchSnapshot()
})