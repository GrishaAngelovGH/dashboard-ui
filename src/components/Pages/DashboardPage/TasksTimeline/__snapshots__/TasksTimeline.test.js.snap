// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="ant-timeline ant-timeline-alternate"
      >
        <li
          class="ant-timeline-item ant-timeline-item-left"
        >
          <div
            class="ant-timeline-item-tail"
          />
          <div
            class="ant-timeline-item-head ant-timeline-item-head-blue"
          />
          <div
            class="ant-timeline-item-content"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, nisi?
          </div>
        </li>
        <li
          class="ant-timeline-item ant-timeline-item-right"
        >
          <div
            class="ant-timeline-item-tail"
          />
          <div
            class="ant-timeline-item-head ant-timeline-item-head-green"
          />
          <div
            class="ant-timeline-item-content"
          >
            Lorem ipsum dolor sit amet.
          </div>
        </li>
        <li
          class="ant-timeline-item ant-timeline-item-left"
        >
          <div
            class="ant-timeline-item-tail"
          />
          <div
            class="ant-timeline-item-head ant-timeline-item-head-custom ant-timeline-item-head-blue"
          >
            <span
              aria-label="clock-circle"
              class="anticon anticon-clock-circle"
              role="img"
              style="font-size: 16px;"
            >
              <svg
                aria-hidden="true"
                data-icon="clock-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
                />
              </svg>
            </span>
          </div>
          <div
            class="ant-timeline-item-content"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur eaque quam, aperiam iste vel ipsum laudantium debitis ratione maxime voluptates sequi reprehenderit repellendus rerum nihil distinctio exercitationem vero inventore magni non. Ipsam ipsum maiores consequatur cupiditate excepturi, neque quam dolor facere quos magnam ea sed maxime expedita vitae nobis aliquam?
          </div>
        </li>
        <li
          class="ant-timeline-item ant-timeline-item-right"
        >
          <div
            class="ant-timeline-item-tail"
          />
          <div
            class="ant-timeline-item-head ant-timeline-item-head-red"
          />
          <div
            class="ant-timeline-item-content"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati, vel!
          </div>
        </li>
        <li
          class="ant-timeline-item ant-timeline-item-left"
        >
          <div
            class="ant-timeline-item-tail"
          />
          <div
            class="ant-timeline-item-head ant-timeline-item-head-blue"
          />
          <div
            class="ant-timeline-item-content"
          >
            Lorem ipsum, dolor sit amet consectetur adipisicing elit. At assumenda id eius. Cupiditate natus reprehenderit sint corrupti dolore provident molestias.
          </div>
        </li>
        <li
          class="ant-timeline-item ant-timeline-item-last ant-timeline-item-right"
        >
          <div
            class="ant-timeline-item-tail"
          />
          <div
            class="ant-timeline-item-head ant-timeline-item-head-custom ant-timeline-item-head-blue"
          >
            <span
              aria-label="clock-circle"
              class="anticon anticon-clock-circle"
              role="img"
              style="font-size: 16px;"
            >
              <svg
                aria-hidden="true"
                data-icon="clock-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
                />
              </svg>
            </span>
          </div>
          <div
            class="ant-timeline-item-content"
          >
            Lorem ipsum dolor sit amet.
          </div>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="ant-timeline ant-timeline-alternate"
    >
      <li
        class="ant-timeline-item ant-timeline-item-left"
      >
        <div
          class="ant-timeline-item-tail"
        />
        <div
          class="ant-timeline-item-head ant-timeline-item-head-blue"
        />
        <div
          class="ant-timeline-item-content"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, nisi?
        </div>
      </li>
      <li
        class="ant-timeline-item ant-timeline-item-right"
      >
        <div
          class="ant-timeline-item-tail"
        />
        <div
          class="ant-timeline-item-head ant-timeline-item-head-green"
        />
        <div
          class="ant-timeline-item-content"
        >
          Lorem ipsum dolor sit amet.
        </div>
      </li>
      <li
        class="ant-timeline-item ant-timeline-item-left"
      >
        <div
          class="ant-timeline-item-tail"
        />
        <div
          class="ant-timeline-item-head ant-timeline-item-head-custom ant-timeline-item-head-blue"
        >
          <span
            aria-label="clock-circle"
            class="anticon anticon-clock-circle"
            role="img"
            style="font-size: 16px;"
          >
            <svg
              aria-hidden="true"
              data-icon="clock-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
              />
              <path
                d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
              />
            </svg>
          </span>
        </div>
        <div
          class="ant-timeline-item-content"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur eaque quam, aperiam iste vel ipsum laudantium debitis ratione maxime voluptates sequi reprehenderit repellendus rerum nihil distinctio exercitationem vero inventore magni non. Ipsam ipsum maiores consequatur cupiditate excepturi, neque quam dolor facere quos magnam ea sed maxime expedita vitae nobis aliquam?
        </div>
      </li>
      <li
        class="ant-timeline-item ant-timeline-item-right"
      >
        <div
          class="ant-timeline-item-tail"
        />
        <div
          class="ant-timeline-item-head ant-timeline-item-head-red"
        />
        <div
          class="ant-timeline-item-content"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati, vel!
        </div>
      </li>
      <li
        class="ant-timeline-item ant-timeline-item-left"
      >
        <div
          class="ant-timeline-item-tail"
        />
        <div
          class="ant-timeline-item-head ant-timeline-item-head-blue"
        />
        <div
          class="ant-timeline-item-content"
        >
          Lorem ipsum, dolor sit amet consectetur adipisicing elit. At assumenda id eius. Cupiditate natus reprehenderit sint corrupti dolore provident molestias.
        </div>
      </li>
      <li
        class="ant-timeline-item ant-timeline-item-last ant-timeline-item-right"
      >
        <div
          class="ant-timeline-item-tail"
        />
        <div
          class="ant-timeline-item-head ant-timeline-item-head-custom ant-timeline-item-head-blue"
        >
          <span
            aria-label="clock-circle"
            class="anticon anticon-clock-circle"
            role="img"
            style="font-size: 16px;"
          >
            <svg
              aria-hidden="true"
              data-icon="clock-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
              />
              <path
                d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
              />
            </svg>
          </span>
        </div>
        <div
          class="ant-timeline-item-content"
        >
          Lorem ipsum dolor sit amet.
        </div>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
